docs/topics/compose-compiler-options.md

```diff
@@ -194,6 +194,27 @@ and functions that are implicitly not skippable (inline functions and functions
 >
 {style="warning"}
 
+### PausableComposition
+
+> The feature flag affects behavior only with a version of Compose runtime that supports pausable composition,
```

Comment:
Why not combine the info and warning into one warning at the top? If the warning is at the bottom of the text then readers can miss it if they don't read through all the content. Or think the warning is applicable to the next section. WDYT? :)